### PR TITLE
Mig document page work instructions to v. 10.0

### PR DESCRIPTION
**Pull request by eugen-don**
Mig document page work instructions to v. 10.0

Depends on: 
- [x] https://github.com/OCA/knowledge/pull/122 
- [x] https://github.com/OCA/management-system/pull/166 (mgtsystem)

### CODE REVIEW

**Comment by dreispt** [comment]
@eugen-don Can you please help reviewing OCA/knowledge#122 ?

File: document_page_procedure/__manifest__.py [hunk 1]
```diff
@@ -20,20 +20,22 @@
 ##############################################################################
 {
     "name": "Document Management - Wiki - Procedures",
-    "version": "9.0.1.0.0",
+    "version": "10.0.0",
```
Comment (dreispt) [inline document_page_procedure/__manifest__.py]: 10.0.1.0.0

File: document_page_procedure/__manifest__.py [hunk 2]
```diff
@@ -20,20 +20,22 @@
...
     "author": "Savoir-faire Linux,Odoo Community Association (OCA)",
     "website": "http://www.savoirfairelinux.com",
     "license": "AGPL-3",
     "category": "Generic Modules/Others",
+    "description": """Procedure Template
+    """,
```
Comment (dreispt) [inline document_page_procedure/__manifest__.py]: Remove description key - we have the README for that.

File: document_page_procedure/__manifest__.py [hunk 3]
```diff
@@ -20,20 +20,22 @@
...
     "depends": [
-        'document_page_work_instruction'
+        'document_page_work_instructions'
```
Comment (dreispt) [inline document_page_procedure/__manifest__.py]: Hmm... module names are supposed to be singular, not plural.
Are you sure about this change?

File: document_page_quality_manual/README.rst [hunk 4]
```diff
@@ -1,39 +1,29 @@
-.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
-    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
-    :alt: License
-
-==============
 Quality Manual
 ==============
 
-This module provides a quality manual template. The template has the same structure as the ISO 9001 standard.
+This module was written to provide a quality manual template. The template has the same structure as the ISO 9001 standard.
```
Comment (dreispt) [inline document_page_quality_manual/README.rst]: IMO the original form is better: we describe what the module does, not how it was done.

File: document_page_quality_manual/README.rst [hunk 5]
```diff
@@ -1,39 +1,29 @@
...
 =====
 
 To enter your quality manual, you need to:
 
-* go to Management Systems > Documentation > Manuals
-* Create a new page and select the Quality Manual category.
-
-.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
-   :alt: Try me on Runbot
-   :target: https://runbot.odoo-community.org/runbot/118/9.0
```
Comment (dreispt) [inline document_page_quality_manual/README.rst]: Why are you removing the Runbot link?

File: document_page_quality_manual/README.rst [hunk 6]
```diff
@@ -42,7 +32,7 @@ Bug Tracker
 Bugs are tracked on `GitHub Issues <https://github.com/OCA/management-system/issues>`_.
 In case of trouble, please check there if your issue has already been reported.
 If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
-`here <https://github.com/OCA/management-system/issues/new?body=module:%20document_page_quality_manual%0Aversion:%209.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+`here <https://github.com/OCA/management-system/issues/new?body=module:%20document_page_quality_manual%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (dreispt) [inline document_page_quality_manual/README.rst]: 8.0?

**Review by dreispt — CHANGES_REQUESTED** [review]
I feel that something went wrong when porting the change history from 9.0 to 10.0.

**Comment by eugen-don** [comment]
Hi @dreispt 
I rebased the pull on latest V 9.0, but im still not familiar with the git magic you mentioned. Is there any template README file i could use?

**Comment by grindtildeath** [comment]
@eugen-don 
Here are the OCA guidelines in case that's what you were looking for : 
https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-10.0
Other wise looks good to me. :+1:

**Review by max3903 — CHANGES_REQUESTED** [review]
@eugen-don In the README.rst, the link to Runbot and Github issues still points or uses version 9.0

**Comment by eugen-don** [comment]
@grindtildeath 
Thanks for providing the resource.

**Review by max3903 — APPROVED** [review]

**Review by sbidoul — APPROVED** [review]

**Comment by max3903** [comment]
@dreispt Can you update your review please?

**Review by dreispt — APPROVED** [review]

**Pull request merged**